ocp_resources/resource.py

```diff
@@ -21,9 +21,10 @@
     NotFoundError,
     ForbiddenError,
 )
+from typing import Any, Dict, List
```

Comment:
@rnetser @myakove I moved `Any, Dict, List` imports since the current code raises this error:
`ImportError: cannot import name 'Any' from 'simple_logger.logger'`

Reply:
@omrirh https://github.com/RedHatQE/openshift-python-wrapper/pull/1864

Reply:
done